### PR TITLE
added placemarker and getplayerchoice

### PR DESCRIPTION
**Pull request by georgej1144**
feel free to review it but we can just take your version.

### CODE REVIEW

File: TicTacToe.cpp [hunk 1]
```diff
@@ -11,31 +9,49 @@ BoxType* CreateBoard()
...
-                cout << 'X';
+                std::cout << 'X';
                 break;
             case (BoxType::O):
-                cout << 'O';
+                std::cout << 'O';
                 break;
             case (BoxType::EMPTY):
-                cout << ' ';
+                std::cout << ' ';
```
Comment (m241dan) [inline TicTacToe.cpp:42]: Should display the number and not a blank so it's easy to see which square they need to input for. (i+1)

Comment (georgej1144) [reply TicTacToe.cpp:42]: I agree, We can just take your implementation. No need to rewrite it imo.

Comment (m241dan) [reply TicTacToe.cpp:42]: Should we just close this then?

Comment (georgej1144) [reply TicTacToe.cpp:42]: Yeah

**Pull request closed without merging**